### PR TITLE
Edits that became larger than intended because I started to drink while I was at it

### PR DESCRIPTION
**Pull request by adswa**
Two things first:
- apologies for the amount of changes. It looks like I butchered the paper up quite a bit (I did so in some places, but in others I have just restructured individual paragraphs). I initially tried to do individual pull requests, but struggled to keep changes small and self-contained.
- I'm not attached to any of the changes. I will not be sad if this PR is closed in its entirety.

I tried to accomplish a few things with this PR.

For one, I tried to increase the coherence within and across paragraphs. I found in particular the summary to be a set of on-their-own-nice-but-not-well-connected sentences. I also tried to attach the subsections within the "Statement of Need" a bit better to each other. Not sure if I made anything better, I started to drink half-way through (sorry). 

I tried to apply the restructuring you settled on in #64.

I gave full vent to my need for harmony and tried to tone down a few phrases which I felt would "point the finger" a bit too much, and tried to rephrase it into something I felt to be less focused on problems but more on solutions.

I tried to extend the section on datalad core a bit.

I didn't get to the last sections of the paper yet, sadly, but wanted to PR as to not hinder others work on this.

Overall, I worked with the current content of the paper, and did not introduce new thoughts - its basically a subjective restructuring and rephrasing.
I don't think this PR is well-suited for review due to its length. Please feel free to close this, or cherry-pick/copy-paste bits and pieces that you like.

### CODE REVIEW

**Comment by yarikoptic** [comment]
Thank you @adswa . We can go with this, if it finds agreement with the others. 
I would just adjust some syntax punctuation and reduce accent on "Python" and "tool" (and may be only slightly re-executability/'run') in summary.

File: paper.md [hunk 1]
```diff
@@ -186,93 +186,76 @@ With this simple approach, git-annex enables separate and optimized implementati
...
-DataLad uses Git's submodule mechanism to unambiguously link (versions of) individual datasets into larger super-datasets, and further simplifies working with the resulting hierarchies of datasets with recursive operations across dataset boundaries.
+Many research datasets contain millions of files, but a large number of files precludes placing datasets in their entirety into a single Git repository even if individual files are tiny in their size.
+Partitioning such datasets into smaller subdatasets (e.g., one subdataset per each subject in the dataset comprising thousands of participants), and linking them seamlessly allows for scalable management.
+Research datasets and projects can also be heterogeneous, comprising different data sources or evolving data across different processing stages.
+Beyond scalability, modularization into homogeneous components also allows for the efficient reuse of a selected subset of datasets.
+<!-- unclear how management of 10k small pieces becomes more scalable than one piece with 10k parts; YOH: just by virtue of being able to handle 10k * 10k files -->
```
Comment (bpoldrack) [inline paper.md]: I think keeping that comment is more confusing than helpful.

File: paper.md [hunk 2]
```diff
@@ -151,24 +151,24 @@ bibliography: paper.bib
...
+Data consumption is not as streamlined, as disconnected data portals require a large variety of different data access and authentication methods;
+Reuse of data and other sizable or binary files is not as transparent, as data versioning is rarely performed, or nowhere close to the precision and inspectability that is standard in software development;
+And scientific processes are not as reproducible, because data provenance, the information of how a digital file came to be, is often incomplete and rarely automatically captured.
+DataLad aims to solve these issues by providing streamlined, transparent management of code, data, computing environments, and their relationship.
+It provides targeted interfaces and interoperability adapters to established scientific and commercial tools and services to set up unobstructed access to all elements of scientific project, and it enables workflows that are particularly suited for reproducible science, such as actionable process provenance capture for arbitrary command execution that affords automatic re-execution.
```
Comment (bpoldrack) [inline paper.md]: Relating to above mentioned varying data access methods, me thinks emphasizing a unified way of access is clearer than just "unobstructed".

```suggestion
It provides targeted interfaces and interoperability adapters to established scientific and commercial tools and services to set up unobstructed, unified access to all elements of scientific project, and it enables workflows that are particularly suited for reproducible science, such as actionable process provenance capture for arbitrary command execution that affords automatic re-execution.
```

**Review by bpoldrack — APPROVED** [review]
Overall LGTM - clear improvement. I don't feel #64 is fully addressed, but the restructuring and rewording does reflect it much better already.

File: paper.md [hunk 3]
```diff
@@ -151,24 +151,24 @@ bibliography: paper.bib
...
+It adapts principles of open-source software development and -distribution to address the technical challenges of data management, data sharing, and digital provenance collection across the life cycle of digital objects:
+By streamlining procedures to consume, publish, and update data, as well as providing means to handle data of any size or type and link them as precisely versioned, lightweight dependencies, DataLad aims to make data management as easy as managing code.
+By capturing complete and actionable process provenance of data transformations, and enabling automatic re-computation based on this provenance, DataLad also helps to make science more reproducible and FAIR [@FAIR2016].
+With a Python API and command-line interface, an infrastructure for software extensions, and complete independence from centralized services but development towards maximum interoperability with existing tools and services, the DataLad project ([datalad.org](http://datalad.org)) delivers a completely open, pioneering platform for flexible decentralized research data management (dRDM) [@Hanke_2021].
```
Comment (bpoldrack) [inline paper.md]: One more additional comment: I never really liked our use of "API" and we use the terms "package" and "library" below anyway. So, I'd rather go with "a python and a command-line interface" in a paper. However, we probably won't get away from "API" due to the module naming, so not too strong a feeling.

```suggestion
With a Python and a command-line interface, an infrastructure for software extensions, and complete independence from centralized services but development towards maximum interoperability with existing tools and services, the DataLad project ([datalad.org](http://datalad.org)) delivers a completely open, pioneering platform for flexible decentralized research data management (dRDM) [@Hanke_2021].
```

**Comment by adswa** [comment]
Thanks for your comments! Maybe @bpoldrack and @yarikoptic could push their changes into this branch?

File: paper.md (hunk 1 above)
Comment (bpoldrack) [reply paper.md]:
```suggestion
```

**Comment by mih** [comment]
Thx @adswa 

I agree that the changes are not debatable in a meaningful way in a PR. I agree with merging this now, and doing additional passes and inclusion of the other proposals #75 #70 in another wave.

**Comment by mih** [comment]
It seems this PR is orthogonal to #70, so I would merge the conclusion of that one first, to give @yarikoptic the chance to respond whether to merge and edit in master, or if he prefers to include fixes directly in this PR. I'd say "merge first", given that #75 also needs changes afterwards.

**Pull request merged**